### ISSUE TITLE
Problem with track-c data

### ISSUE THREAD

**Issue by ammar-yasser1** [emotion-analysis-project/SemEval2025-Task11#6]
there is no training data and the dev data is Nan so i can't fine tune a model using this data, can i use another dataset or what can i do?

**Comment by nedjmaou** [comment]
The training and dev data are track A's. For languages with no training data (or those whose training data will be released), you will have a test set when the evaluation period starts.

External datasets are allowed. But as explained in the track C's description, the source language of the data has to be different from the target language.
